input/fsh/SD_Extensions.fsh

```diff
@@ -59,9 +63,38 @@ Description: "Dose parameters prescribed for one radiotherapy volume."
   * ^short = "Number of Prescribed Fractions"
   * ^definition = "The number of Fraction in which dose to this volume is prescribed. See also extension RadiotherapyFractionsPrescribed which is used instead if fractions are not per volume, e.g. in Phase Prescriptions or Plans."
 
+Extension: RadiotherapyDosePlannedToVolume
+Id: codexrt-radiotherapy-dose-planned-to-volume
+Title: "Dose Planned to Volume"
+Description: "Dose parameters planned for one radiotherapy volume."
+* . ^short = "Planned Dose to a Dose Reference"
+* extension contains
+    volume 1..1 MS and
+    totalDosePlanned 0..1 MS and
+    fractionsPlanned 0..1 MS and
+    fractionDosePlanned 0..1 MS
+* extension[volume]
+  * value[x] only Reference(RadiotherapyVolume)
+  * ^short = "Volume to which Dose is Planned"
+  * ^definition = "A BodyStructure resource representing the body structure treated, for example, Chest Wall Lymph Nodes."
```

Comment:
treated --> to be treated, or that is the target of treatment

Reply:
> treated --> to be treated, or that is the target of treatment
Fixed to "to be treated".